packages/component-library-vue/src/UtrechtTextarea.vue

```diff
@@ -1,32 +1,34 @@
-<script setup lang="ts">
-  defineEmits<(_type: "update:modelValue", _value: unknown) => void>();
+<script lang="ts">
+  import { defineComponent } from "vue";
+  import { useModelWrapper } from "./helpers/modelWrapper";
 
-  defineProps<{
-    disabled?: boolean;
-    invalid?: boolean;
-    modelValue?: string;
-    readonly?: boolean;
-    required?: boolean;
-    type?: string;
-  }>();
+  export default defineComponent({
+    props: {
+      invalid: { type: Boolean, required: false },
+      readonly: { type: Boolean, required: false },
+      options: { type: [Array, Object], required: true },
+      modelValue: { type: [String, Number, Boolean], require: false, default: "" },
+    },
+    setup(props, { emit }) {
+      return {
+        value: useModelWrapper(props, emit),
+      };
+    },
+  });
 </script>
 
 <template>
   <textarea
-    class="utrecht-textarea"
+    v-model="value"
     :aria-invalid="invalid || undefined"
-    :class="{
-      'utrecht-textarea--disabled': disabled,
-      'utrecht-textarea--invalid': invalid,
-      'utrecht-textarea--required': required,
-      'utrecht-textarea--readonly': readonly,
-    }"
-    :disabled="disabled"
+    :class="[
+    { 'utrecht-textarea--invalid': invalid },
+    { 'utrecht-textarea--readonly': readonly },
+    'utrecht-textarea',
+    'utrecht-textarea--html-textarea'
+    ]"
     :readonly="readonly"
```

Comment:
Is `readonly` still necessary to implement explicity, or is it also covered by `utrecht-textarea--html-textarea`?